Bendy and the Dark Revival/BATDR.Settings.xml

```diff
@@ -29,6 +29,8 @@
         <Setting Id="obj_11201" Label="GentPipeUpgrade" State="false" />
         <Setting Id="obj_11202" Label="GentPipeCharge" State="false" />
         <Setting Id="obj_11301" Label="Elevators" State="false" />
+        <Setting Id="kw" Label="KingWidow" State="false" ToolTip="Splits after the fight, when talking to Alice in the Pipe." />
```

Comment:
This can just be `csp_11503` directly, you don't need to use the settings aliases. Same goes for `mww` and `dw`.